sympy/combinatorics/free_groups.py

```diff
@@ -140,30 +139,27 @@ class FreeGroup(DefaultPrinting):
 
     def __new__(cls, symbols):
         symbols = tuple(_parse_symbols(symbols))
-        rank = len(symbols)
-        _hash = hash((cls.__name__, symbols, rank))
-        obj = _free_group_cache.get(_hash)
-
-        if obj is None:
-            obj = object.__new__(cls)
-            obj._hash = _hash
-            obj._rank = rank
-            # dtype method is used to create new instances of FreeGroupElement
-            obj.dtype = type("FreeGroupElement", (FreeGroupElement,), {"group": obj})
-            obj.symbols = symbols
-            obj.generators = obj._generators()
-            obj._gens_set = set(obj.generators)
-            for symbol, generator in zip(obj.symbols, obj.generators):
-                if isinstance(symbol, Symbol):
-                    name = symbol.name
-                    if hasattr(obj, name):
-                        setattr(obj, name, generator)
-
-            _free_group_cache[_hash] = obj
+        obj = super().__new__(cls, Tuple(*symbols))
+        obj._symbols = symbols
+        obj._generators = None
+        for symbol, generator in zip(obj.symbols, obj.generators):
+            if isinstance(symbol, Symbol):
+                name = symbol.name
+                if hasattr(obj, name):
+                    setattr(obj, name, generator)
```

Comment:
This seems flakey. What if I have `Symbol('_args')`?

Reply:
I see the coverage indicates that this is not used and originally implemented in a wrong way. So I'll drop this completely.